### DISCUSSION TITLE
Welcome to Lailloken-UI Discussions!

### DISCUSSION THREAD

**Discussion by Lailloken** [Lailloken/Exile-UI#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Esgarrouth** [comment]
Hello. Im using 2 monitors and the overlay shows up on the wrong screen. I cant move it like I do normal windows. Any idea how to fix this?

**Reply by Lailloken** [reply]
I'll probably start working on multi-monitor support today once I've dug up my old monitor for testing. In the meantime you have to switch your system settings around. The overlay currently only appears on the monitor that is set as the primary in your system settings, so you need to set whichever monitor you want the overlay on to primary.

**Reply by Esgarrouth** [reply]
Ok, thanks for the info. Keep up the good work.
